src/runtime/description-formatter.ts

```diff
@@ -456,7 +456,10 @@ export class DescriptionFormatter {
       // Choose connections with patterns (manifest-based or dynamic).
       const connectionSpec = connection.spec;
       const particleDescription = this.particleDescriptions.find(desc => desc._particle === connection.particle);
-      return !!connectionSpec.pattern || !!particleDescription._connections[connection.name].pattern;
+      // TODO(sjmiles): added particleDescription null-check for
+      // the moment, but we need to root cause this problem
+      return !!connectionSpec.pattern ||
```

Comment:
just a shot in the dark - maybe this is because the new`createForPlan` method doesn't include the inner arc's particles?